src/os/flatpak.zig

```diff
@@ -3,6 +3,7 @@ const assert = std.debug.assert;
 const Allocator = std.mem.Allocator;
```

Comment:
This appears to be a duplicate commit from #6658 . Should rebase so that this PR only includes the resources dir change.

Reply:
I can't do much until #6658 is merged.